rust/agama-cli/Cargo.toml

```diff
@@ -18,10 +18,11 @@ tempfile = "3.13.0"
 nix = { version = "0.27.1", features = ["user"] }
 tokio = { version = "1.40.0", features = ["macros", "rt-multi-thread"] }
 async-trait = "0.1.83"
-reqwest = { version = "0.11", features = ["json"] }
+reqwest = { version = "0.12.8", features = ["json"] }
```

Comment:
BTW this was the only case where we required 2 different versions of a library.
I wonder if there is an automated check for such things

Reply:
Thanks, I did not know `cargo tree`. It does not do exactly what I want but it is a good start